src/vis/vis.js

```diff
@@ -142,9 +142,18 @@ var VisModel = Backbone.Model.extend({
 
     var windshaftClient = new WindshaftClient(windshaftSettings);
 
+    // Create the public Analysis Factory
+    this.analysis = new AnalysisFactory({
```

Comment:
IMO `this.analysisFactory` would be a more descriptive name

Reply:
just realized that the AnalysisFactory is not a factory (software pattern) maybe we should look for a better name in the future

Reply:
Agree and agree. Any suggestions? `AnalysisService`?

Reply:
AnalysisService will do the work!

Reply:
Ok. `vis.analysis` is part of the API that BUILDER is using now so I wouldn't rename it for now (given that the API is going to change anyway). I also agree about renaming `AnalysisFactory` to `AnalysisService` but let's do it in a separate PR to keep this one small.